### PR TITLE
Feature/implement fn:contains-token

### PR DESCRIPTION
**Pull request by tuurma**
implements fn:contains-token with relevant tests

### CODE REVIEW

File: exist-core/src/test/xquery/xquery3/fn.xql [hunk 1]
```diff
@@ -122,3 +122,37 @@ function fnt:outermost() {
             ","
         )
 };
+
+declare
+      %test:args('red green blue ', 'red')          %test:assertTrue
+      %test:args("red, green, blue", ' red ')       %test:assertFalse
+      %test:args('', 'red')                         %test:assertFalse
```
Comment (duncdrum) [inline exist-core/src/test/xquery/xquery3/fn.xql]: can we add a test for `empty-sequence` here, i don't think that is the same as `' '` the specs say this should be false, so we shouldn't see a type error. For future maintainability i would also add a link to the relevant [spec page](https://www.w3.org/TR/xpath-functions-31/#func-contains-token) in an xqdoc comment

Comment (tuurma) [reply exist-core/src/test/xquery/xquery3/fn.xql]: @duncdrum https://docs.google.com/document/d/1xPZEa-dCcwt5vqZlTQR2oVc9A_sXdkvyCmWYVFxAKK0/edit# there already is a test for empty input sequence or did you mean for the token too?

Comment (duncdrum) [reply exist-core/src/test/xquery/xquery3/fn.xql]: Is empty string `””` the same as empty sequence ‘()’? If so all is good

Comment (tuurma) [reply exist-core/src/test/xquery/xquery3/fn.xql]: @duncdrum fnt:contains-token-tests-empty-input-sequence already tests the empty input sequence, see l.149. Thanks for spotting the ;;

**Comment by tuurma** [comment]
just created documentation PR and ported to 5.0.0
